src/framework/xr/xr-anchors.js

```diff
@@ -380,8 +388,24 @@ class XrAnchors extends EventHandler {
      * @ignore
      */
     update(frame) {
-        if (!this._available)
+        if (!this._available) {
+            // enabledFeatures - is not available, requires alternative way to check feature availability
+            if (!this.manager.session.enabledFeatures && !this._checkingAvailability) {
+                this._checkingAvailability = true;
+
+                frame.createAnchor(new XRRigidTransform(new Vec3(), new Quat()), this.manager._referenceSpace) // eslint-disable-line no-undef
```

Comment:
I'm guessing Intellisense in VS Code is going to complain that you're passing in a `Vec3` and `Quat` here. But as it happens, the `XRRigidTransform` constructor parameters are optional and if you leave them unspecified, the equivalent effect will be made. So consider just deleting them (and the imports).

Reply:
Also, instead of specifying `eslint-disable-line no-undef`, it's probably better to add `XRRigidTransform` to the ESLint ignore list in `package.json`.